webui/src/Components/TabArea.css

```
@@ -0,0 +1,171 @@
+		color: var(--color-tab-fg);
+		background-color: var(--color-tab-bg);
+		font-weight: 500;
+		text-decoration: none;
+		border: 1px solid var(--color-border);
+		border-bottom-width: 0;
+		border-top-left-radius: 6px;
+		border-top-right-radius: 6px;
+		cursor: pointer;
+		appearance: none;
+		-webkit-appearance: none;
+		font: inherit;
```

Stylelint (17.14.0)

[error] Overridden property "font-weight" by shorthand "font" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)